packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-phone/java/integrationsteps.md

```diff
@@ -1,3 +1,100 @@
+
+> **Note:** If the user already has the phone authenticator enrolled, then `AuthenticationStatus=AWAITING_AUTHENTICATOR_SELECTION` is returned (instead of `AuthenticationStatus=AWAITING_AUTHENTICATOR_ENROLLMENT_SELECTION`) and the user does not have to enroll the phone authenticator with a phone number, bypassing steps [3](#step-3-user-selects-phone-authenticator) and [4](#step-4-user-enters-phone-number).
+
+After receiving the `AWAITING_AUTHENTICATOR_ENROLLMENT_SELECTION` status and the list of authenticators to be enrolled, provide the user with a form to select the authenticator to enroll. In the following example, phone is the only authenticator:
+
+<div class="common-image-format">
+
+![Displays the enroll phone authenticator selection page for Java SDK](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-sign-in-pwd-phone-enroll-phone-java.png)
+
+</div>
+
+> **Tip:** Build a generic authenticator selection form to handle single or multiple authenticators returned from the SDK.
+
+### Step 3: User selects phone authenticator
+
+In this use case, the user selects **Phone** as the authenticator to enroll. Pass this selected authenticator to the `IDXAuthenticationWrapper.selectAuthenticator()` method.
+
+```java
+authenticationResponse = idxAuthenticationWrapper.selectAuthenticator(proceedContext, authenticator);
+```
+
+The response from this request is an `AuthenticationResponse` object with `AuthenticationStatus=AWAITING_AUTHENTICATOR_ENROLLMENT_DATA`. This status indicates that the user needs to provide additional authenticator information. In the case of the phone authenticator, the user needs to specify a phone number.
+
+> **Note:** Only SMS is currently supported for the phone authenticator method factor. In the future, if your org supports the voice factor, you will need to add a voice or an SMS factor selection form for your app. Both the phone authenticator and the phone method factor needs to be passed to the [`IDXAuthenticationWrapper.submitPhoneAuthenticator()`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java#L398) method.
+
+You need to build a form to capture the user's phone number in your app. For example:
+
+<div class="common-image-format">
+
+![Displays the Java SDK's phone number input form](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-in-pwd-phone-numberscreen-java.png)
+
+</div>
+
+> **Note:** The Java SDK requires the following phone number format: `{+}{country-code}{area-code}{number}`. For example, `+15556667777`.
+
+### Step 4: User enters phone number
+
+When the user submits their phone number, capture this information and pass it to the [`IDXAuthenticationWrapper.verifyAuthenticator()`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java#L368) method. In the following code example, the phone number is saved in the `code` variable:
```

Comment:
The fact that we need this sentence is kinda sad. It's a shortcoming of the SDK. We're trying to accomplish 2 things, dynamic nature of the API, the static ("easy") nature of the SDK.

The SDK doesn't differentiate these uses cases between code/phone number etc. And I think it's pretty confusing. I feel like we should leave this as feedback for the SDK, and potentially do some enhancements to make this more clear.

Reply:
I will remove the line explaining that the phone number is in `code`.  I'll just tell the developer to put the phone number in the `VerifyAuthenticatorOptions` object. Hopefully that will avoid the sad confusion.